pom.xml

```diff
@@ -1010,10 +1010,15 @@
                                     --> -Xep:AndroidJdkLibsChecker:OFF <!--
                                     We don't target JDK 7.
                                     --> -Xep:Java7ApiChecker:OFF <!--
+                                    We don't target JDK 8.
+                                    --> -Xep:Java8ApiChecker:OFF <!--
                                     Deals with an Android-specific limitation
                                     not applicable to us. See also
                                     https://github.com/google/error-prone/issues/488.
                                     --> -Xep:StaticOrDefaultInterfaceMethod:OFF <!--
+                                    XXX: This check flags false positives.
+                                    See https://github.com/google/error-prone/issues/2679.
+                                    --> -Xep:VoidMissingNullable:OFF <!--
```

Comment:
It's a bit weird how the `-->` and `<--` are on the same line as the checks are disabled. We don't have that internally, is there a specific reason for that?

Reply:
See [this documentation](https://github.com/google/error-prone/commit/c8f2c177d949f0984022730240fe17668d8845fe#diff-3981efdc8e2553d38238838106f5859798b9f590e3d453779d7d9c56d7f0d14dR132-R144). This formatting is a remnant of our support for JDK 8. I plan to open one or more PRs to sync internal Maven parent improvements to this repo; will then also clean this up.